eng/helix/content/RunTests/RunTests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
This won't work. The property is not defined on Helix agents and that's where this project is built.